packages/block-library/src/template-part/index.php

```diff
@@ -19,12 +19,14 @@ function render_block_core_template_part( $attributes ) {
 	$content          = null;
 	$area             = WP_TEMPLATE_PART_AREA_UNCATEGORIZED;
 
-	if (
-		isset( $attributes['slug'] ) &&
-		isset( $attributes['theme'] ) &&
-		get_stylesheet() === $attributes['theme']
-	) {
-		$template_part_id    = $attributes['theme'] . '//' . $attributes['slug'];
+	if ( isset( $attribues['theme'] ) ) {
+		$theme = $attributes['theme'];
+	} else {
+		$theme = get_stylesheet();
+	}
+
+	if ( isset( $attributes['slug'] ) ) {
```

Comment:
We probably want to keep this condition to not change the behavior:
```suggestion
	if ( isset( $attributes['slug'] ) && get_stylesheet() === $theme ) {
```